src/web/services/feedback-responses.service.ts

```diff
@@ -119,6 +122,24 @@ export class FeedbackResponsesService {
     }
   }
 
+  /**
+   * Retrieves a feedback response by calling API.
+   */
+  getFeedbackResponse(queryParams: {
+    questionId: string,
+    intent: Intent,
+    key: string,
+    moderatedPerson: string,
+  }): Observable<FeedbackResponsesResponse> {
+    const paramMap: { [key: string]: string } = {
+      questionid: queryParams.questionId,
+      intent: queryParams.intent,
```

Comment:
Nitpicking I guess, but maybe ES6 destructuring can come into handy here, so that you can just do 
```
// do destructuring here
// then can simplify this part
const paramMap: { [key: string]: string } = {
   questionid 
   intent,
   key,
   moderatedperson
}
```